### ISSUE TITLE
Deprecated Dependancies

### ISSUE THREAD

**Issue by ArlinJae** [ScottyFillups/giorgio#1]
```
npm warn deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm warn deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm warn deprecated har-validator@5.1.5: this library is no longer supported
npm warn deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm warn deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm warn deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm warn deprecated fsevents@1.2.13: The v1 package contains DANGEROUS / INSECURE binaries. Upgrade to safe fsevents v2
npm warn deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

**Comment by philipjscott** [comment]
how did you find this repo?

**Comment by ArlinJae** [comment]
https://student.cs.uwaterloo.ca/~se212/george/george-docs-1/contributions.html

---
Jai Relan
jairelan.com ( http://jairelan.com/ )
LinkedIn ( https://www.linkedin.com/in/jairelan/ ) | Twitter ( https://twitter.com/JaiRelan )

Sent via Superhuman iOS ( ***@***.*** )

**Comment by ArlinJae** [comment]
I’m guessing the API endpoint that your cli is pinging is probably broken, right? Taking 212 this sem and I’m not a huge fan of using the online compiler. I’d love for a way to execute George through the CLI

---
Jai Relan
jairelan.com ( http://jairelan.com/ )
LinkedIn ( https://www.linkedin.com/in/jairelan/ ) | Twitter ( https://twitter.com/JaiRelan )

Sent via Superhuman iOS ( ***@***.*** )

**Comment by philipjscott** [comment]
Ah, you're a UW student. I saw `El Segundo, California` and thought you were just a random person.

You're the first 2x founder | investor I've bumped into doing 212. So when are you getting partner at Sequoia?

I'll fix this up tonight. Gonna fire off a LinkedIn invite if you don't mind, I'm SE2021 and YCW22 and we might have some stuff in common.

**Comment by ArlinJae** [comment]
yeah man, I don't think random people would be looking at George CLI runtimes in their free time ://

You're good man, happy to take this over if you want - don't want to add another thing to your plate. btw, if you want, down to jump on a call and chat sometime! We're connected on Li so we can coordinate specs there

**Comment by philipjscott** [comment]
@jairelan https://github.com/philipjscott/george-cli/tree/master

try it out ^

**Comment by ArlinJae** [comment]
Worked now man, thank you so much! Appreciate you bringing this repo out of the dead... 

Are you still based out of KW? Let's grab coffee

**Comment by philipjscott** [comment]
@jairelan Not in KW atm, but I'll be sure to ping you when I'm in the area and grab a coffee :)

I created an entirely new repo since this repo's code is mid. The new repo has some cool new features, like being able to test multiple `grg` files in one go and combine the output cleanly. Watch mode won't output anything if the file hasn't changed.

Say hi to Patrick Lam and Derrick Rayside for me — I hope they're doing well. I'd also appreciate you starring the new repo and telling people about it

**Comment by philipjscott** [comment]
I'll close this issue later tonight once I fix this package and change the README to tell people to use `george-cli`. Cheers!
